website/docs/r/org_group.html.markdown

```diff
@@ -0,0 +1,84 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_org_group"
+sidebar_current: "docs-vcd-resource-org-group"
+description: |-
+  Provides a vCloud Director Organization group. This can be used to create, update, and delete organization groups defined in SAML or LDAP.
+---
+
+# vcd\_org\_group
+
+Provides a vCloud Director Organization group. This can be used to create, update, and delete
+organization groups defined in `SAML` or `LDAP`.
+
+Supported in provider *v2.9+*
+
+~> **Note:** This operation requires the rights included in the predefined `Organization
+Administrator` role or an equivalent set of rights. `SAML` or `LDAP` must be configured as vCD
+does not support local groups and will return HTTP error 403 "This operation is denied." if selected
+`provider_type` is not configured.
+
+## Example Usage
+
+```hcl
+resource "vcd_org_group" "org1" {
+  org  = "org1"
+  
+  provider_type = "SAML"
+  name          = "Org1-AdminGroup"
+  role          = "Organization Administrator"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to which the VDC belongs. Optional if defined at provider level.
+* `name` - (Required) A unique name for the group.
+* `description` - (Optional) The description of Organization group
```

Comment:
Maybe add Note that only works for LDAP

Reply:
It works with SAML as well. Just update doesn't work. If one sets it during create - it works. It is a bug in vCD so not sure if we should put it here as at some point in time it will become invalid.

cc @dataclouder to give 3rd vote :)